lib/client/metrics/metrics.js

```diff
@@ -0,0 +1,42 @@
+const metrics = require('../../middleware/metrics/metrics')
+
+let metricsClient
+
+const init = () => {
+  if (metricsClient) {
+    return
+  }
+  const prometheusClient = metrics.getClient()
+
+  // values for metrics
+  const labelNames = ['client_name', 'base_url', 'url', 'method', 'status_code', 'status_message', 'error_name']
+  const buckets = [0.1, 0.25, 0.5, 1, 2, 5, 10]
+
+  const apiMetrics = new prometheusClient.Histogram({
+    name: 'jwt_api_requests',
+    help: 'API calls using FB JWT Client',
+    labelNames,
+    buckets
+  })
+
+  const requestMetrics = new prometheusClient.Histogram({
+    name: 'jwt_client_requests',
+    help: 'HTTP requests using FB JWT Client',
+    labelNames,
+    buckets
+  })
+
+  metricsClient = {
+    apiMetrics,
+    requestMetrics
+  }
+}
+
+const getMetricsClient = () => {
+  init()
```

Comment:
Wonder if there are any side effects of putting this init within a getter

Reply:
LIke what?

Reply:
Was more just flagging as a potential thing, if at all, as there are ` new prometheusClient.Histogram()` things happening within a getter, but maybe that's fine!

Reply:
If you fancy adding a test that proves that the meat of the init function only ever gets run once, feel free ;)